e2e-tests/playwright/e2e/plugins/custom-theme/custom-theme.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { test, Page, TestInfo } from '@playwright/test';
+import { Common, setupBrowser } from '../../../utils/Common';
+import { ThemeVerifier } from './theme-verifier';
+
+let page: Page;
+
+test.describe('CustomTheme should be applied', () => {
```

Comment:
by test steps it looks like its serial test case, can we make it serial using `test.describe.serial()`